code/datums/gamemode/role/vampire_role.dm

```diff
@@ -166,76 +167,29 @@
 			to_chat(assailant, "<span class='notice'>You have accumulated [blood_total] [blood_total > 1 ? "units" : "unit"] of blood[blood_usable_before != blood_usable ?", and have [blood_usable] left to use." : "."]</span>")
 		check_vampire_upgrade()
 		target.vessel.remove_reagent(BLOOD,25)
+		update_vamp_hud()
```

Comment:
We now have a single antag hud intoduced by @DeityLink , use that.

Reply:
This is for the blood of the vamp. The antag hud of Deity is more about who's in your team.

Reply:
o ok